lib/printer.js

```diff
@@ -954,6 +1006,9 @@ function genericPrintNoParens(path, options, print) {
     case "Specifier":
     case "NamedSpecifier":
     case "Comment": // Supertype of Block and Line.
+    case "MemberTypeAnnotation": // Flow
+    case "TupleTypeAnnotation": // Flow
+    case "Type": // Flow
```

Comment:
I couldn't find these three types anywhere. I assume "Type" is just the supertype in ast-types; `TupleTypeAnnotation` is implicit (?) and I don't know what `MemberTypeAnnotation` is – it is not even in esprima, so I should probably just get rid of it here.